tests/ops/integration_tests/test_integration_email.py

```diff
@@ -0,0 +1,141 @@
+                "id": 1,
+                "name": "John Customer",
+                "email": "customer-1@example.com",
+                "address_id": 1,
+            }
+        ],
+        "postgres_example_test_dataset:report": [],
+        "postgres_example_test_dataset:address": [],
+        "postgres_example_test_dataset:employee": [],
+        "postgres_example_test_dataset:login": [],
+        "postgres_example_test_dataset:orders": [],
+        "postgres_example_test_dataset:order_item": [],
+        "postgres_example_test_dataset:payment_card": [],
+        "postgres_example_test_dataset:product": [],
+        "postgres_example_test_dataset:service_request": [],
+        "postgres_example_test_dataset:visit": [],
+        "email_dataset:daycare_customer": [],
+        "email_dataset:children": [],
+        "email_dataset:payment": [],
+    }
+
+    dataset_postgres = FidesopsDataset(**example_datasets[0])
+    dataset_email = FidesopsDataset(**example_datasets[9])
+    postgres_graph = convert_dataset_to_graph(
+        dataset_postgres, integration_postgres_config.key
+    )
+    email_graph = convert_dataset_to_graph(dataset_email, email_connection_config.key)
+    dataset_graph = DatasetGraph(*[postgres_graph, email_graph])
+
+    v = graph_task.run_erasure(
+        privacy_request,
+        erasure_policy,
+        dataset_graph,
+        [integration_postgres_config, email_connection_config],
+        {"email": "customer-1@example.com"},
+        cached_data_for_erasures,
+        db,
+    )
+
+    assert v == {
+        "email_dataset:payment": 0,
+        "postgres_example_test_dataset:customer": 0,
+        "postgres_example_test_dataset:report": 0,
+        "postgres_example_test_dataset:employee": 0,
+        "postgres_example_test_dataset:service_request": 0,
+        "postgres_example_test_dataset:visit": 0,
+        "postgres_example_test_dataset:orders": 0,
+        "postgres_example_test_dataset:login": 0,
+        "postgres_example_test_dataset:address": 0,
+        "postgres_example_test_dataset:payment_card": 0,
+        "email_dataset:daycare_customer": 0,
+        "postgres_example_test_dataset:order_item": 0,
+        "email_dataset:children": 0,
+        "postgres_example_test_dataset:product": 0,
+    }, "No data masked by Fidesops for the email collections"
+
+    raw_email_template_values = (
+        privacy_request.get_email_connector_template_contents_by_dataset(
+            CurrentStep.erasure, "email_dataset"
+        )
+    )
+
+    assert raw_email_template_values == {
+        "children": CollectionActionRequired(
+            step=CurrentStep.erasure,
+            collection=CollectionAddress("email_dataset", "children"),
+            action_needed=[
+                ManualAction(
+                    locators={
+                        "parent_id": ["email_dataset:daycare_customer:id"]
+                    },  # The only locator is on a separate collection on their end. We don't have data for it.
```

Comment:
It is tricky if one email collection is dependent on another, we won't have any data to give them to query, they need to locate on their own.  Here, we don't have any `parent_ids` to give them, they need to go to the daycare_customer collection, use the customer_id of 1, and then find the parent_id, and use that here.

Reply:
Interesting edge case! @mfbrown do you have any insight on whether this is likely to happen?